### PR TITLE
Fix broken blog social images

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

File: src/templates/blog-post.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
         image={
           picture &&
           (isProduction
-            ? `/blog/${picture.childImageSharp.fields.sourcePath}`
+            ? `/blog/images/${picture.childImageSharp.fields.sourcePath}`
```
Comment (julieg18) [inline src/templates/blog-post.tsx:72]: The `sourcePath` at some point changed, breaking how we generated our social image paths. Maybe when #3396 was merged, but I'd need to debug further to be sure.

**Review by shcheklein — APPROVED** [review]

**Comment by shcheklein** [comment]
CSS lint tests are failing ...

**Review by shcheklein — APPROVED** [review]

**Comment by julieg18** [comment]
> CSS lint tests are failing ...

Yes, both CSS and JS lints tests are failing right now! Investing further right now but it doesn't have anything to do with this pr.

File: src/templates/blog-post.tsx (hunk 1 above)
Comment (rogermparent) [reply src/templates/blog-post.tsx:72]: Pretty sure it was 3396, I remember having to change something to do with sourcepath in order to fix a build-breaking issue

**Pull request merged**